### ISSUE TITLE
Avatar - missing text equivalent for image/canvas function 

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#717]
### Expected behavior

Text communicates function of being able to upload/change an image for the avatar. 
### Actual behavior

There is no text description of this function.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.0](https://github.com/blackbaud/skyux/releases/tag/v1.7.0)
